dashing/ci-nightly-performance.yaml

```diff
@@ -184,5 +185,81 @@ show_plots:
       selection_flag: INCLUDE_BY_COLUMN
       selection_value: 5
       url: /job/Dci__nightly-performance_ubuntu_bionic_amd64/%build%/artifact/ws/test_results/buildfarm_perf_tests/performance_test_results_%name%_cpu_usage.png
-
+  Performance Two Processes Test Results:
+  - title: Average Round-Trip Time
+    y_axis_label: Milliseconds
+    master_csv_name: plot-5516033016820989579.csv
+    style: line
+    num_builds: 10
+    y_axis_minimum: 0
+    y_axis_maximum: 0.2
+    y_axis_exclude_zero: False
+    data_series:
+    - data_file: ws/test_results/buildfarm_perf_tests/performance_test_two_process_results_*.csv
+      data_type: csv
+      selection_flag: INCLUDE_BY_COLUMN
+      selection_value: 0
+      url: /job/Dci__nightly-performance_ubuntu_bionic_amd64/%build%/artifact/ws/test_results/buildfarm_perf_tests/performance_test_two_process_results_%name%.png
```

Comment:
Wouldn't it make sense to only specify relative paths in the config file and let the absolute URL be computed by the Python logic? That would avoid having to embed the job name here which is redundant to the filename.

Reply:
None of our python logic touches this string. It is passed directly to the plot plugin, which is displayed at a URL that would require `..` to be resolved to get to this particular URL.

I don't see how we could make it happen...

Reply:
> I don't see how we could make it happen...

The Python logic already reads the value from `yaml` (https://github.com/ros-infrastructure/ros_buildfarm/blob/3550bcc4c2dbdd62e852a86e51e82687e6a66c7b/ros_buildfarm/config/plot_config.py#L93). There or wherever it is being used afterwards it can be modified as needed.

Reply:
So we should guess the URL at which the plot will be displayed in Jenkins and combine it with a relative path given here?

How is that not more fragile than what we're doing now?

Reply:
Which part do we have to guess?

Reply:
Now that I've actually looked at it, I'll admit that this isn't as bad as I initially thought. I was under the impression that the individual plot groups were under their own URLs, but it looks like the plugin just dumps everything directly under `/plot/`.

So we'd have to make an assumption that the plot plugin's URL scheme doesn't change, and we'd also have to make an assumption whether the user is viewing the plot page under a specific Jenkins view or not (clearly we can't know this at job generation time, so likely default to "no view").

It's feasible.